sonos/__init__.py

```diff
@@ -3007,12 +3007,12 @@ def __init__(self, sh):
             self._tts = self.get_parameter_value("tts")
             self._snippet_duration_offset = float(self.get_parameter_value("snippet_duration_offset"))
             self._discover_cycle = self.get_parameter_value("discover_cycle")
-            self.webif_pagelength = self.get_parameter_value('webif_pagelength')
             local_webservice_path = self.get_parameter_value("local_webservice_path")
             local_webservice_path_snippet = self.get_parameter_value("local_webservice_path_snippet")
             webservice_ip = self.get_parameter_value("webservice_ip")
             webservice_port = self.get_parameter_value("webservice_port")
             speaker_ips = self.get_parameter_value("speaker_ips")
+            self._pause_item_path = self.get_parameter_value('pause_item')
         except KeyError as e:
```

Comment:
Ist keine deiner Änderungen, aber wenn die Parameter in plugin.yaml definiert sind, sind sie immer vorhanden, wenn sie required oder defaults vorgegeben sind.

Reply:
so ist die aktuelle Definition:

```
  pause_item:
    type: str
    default: ''
    description:
      de: 'Item, um die Ausführung des Plugins zu steuern'
      en: 'item for controlling plugin execution'

```
Somit muss das Plugin auch laufen, wenn der parameter nicht gesetzt ist.

Reply:
@Morg42 
Schau mal in Zeile 3059. Ich glaube da liegt der Fehler.

Es muss dort sicher heißen: 
`if self._pause_item_path:`

Reply:
Und zu "muss auch laufen" - die Korrekturen in SmartPlugin sind erst im 1.10er branch vorhanden, mit 1.9.x gibt es Fehler.